public/main.js

```diff
@@ -136,4 +137,16 @@ function deleteTodo(todoId, callback) {
     deleteRequest.send();
 }
 
+function deleteCompleted() {
+    var promise = new Promise(function(resolve, reject) {
+        todosLocal.forEach(function(todo) {
+            if (todo.isComplete) {
+                deleteTodo(todo.id);
+            }
+        });
+    });
+    promise.then(reloadTodoList);
```

Comment:
I originally tried doing the refresh each time but it was giving me funky lists. I'll retry it.